addons/web/static/src/search/favorite_menu/favorite_menu.xml

```diff
@@ -8,7 +8,7 @@
                 <span class="o_dropdown_title">Favorites</span>
             </t>
             <t t-set="currentGroup" t-value="null"/>
-            <t t-foreach="items" t-as="item" t-key="item.id or item.key">
+            <t t-foreach="this.items" t-as="item" t-key="item.id or item.key">
```

Comment:
For commit: "(because the favorite are store in the view arch, and not with
ir.filters records)." -> "because favorites are stored in view arc and not using ir.filters records" ?